pkg/node/node.go

```diff
@@ -85,6 +86,7 @@ type Node struct {
 	ComputeNode    *Compute
 	RequesterNode  *Requester
 	CleanupManager *system.CleanupManager
+	LicenseManager *licensing.LicenseManager
```

Comment:
We don't need to store the manager in the node struct

Reply:
@wdbaruni I added it to the node since it will be a "Node Wide" activity to check for the license. I am assuming that the license manager will be used in many places in the node, so making it easier on the caller side.

I can remove it, but each caller that is not instantiated in the NewNode needs to figure out how to get this manager.

Reply:
The fields we keep and expos at the node level are mainly for testing purposes and to make them accessible during certain tests. We should minimize those instances and just use the Node to instantiate and inject dependencies

Reply:
LicenseManager removed from Node.